flink-external-resource/flink-external-resource-gpu/src/test/resources/test-gpu-discovery.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
I would call this `testing-gpu-discovery.sh`.
My understanding is that, `test-xyz` is usually some test cases that verifies the behavior of `xyz`, while `testing-xyz` is some special implementation of `xyz` for testing purpose.